CHANGES.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.10.17 (core) / 0.26.17 (libraries)
+
+### Bugfixes
+- Fixed an issue where run logs could not be displayed.
```

Comment:
Maybe "Fixed an issue where an error was displayed in the UI while viewing run logs".